plugins/python/redisgears_python.c

```diff
@@ -66,17 +66,19 @@ static int PyInitializeRedisAI(){
         return REDISMODULE_OK;
     }
 
-    RedisModuleCtx* ctx = RedisModule_GetThreadSafeContext(NULL);
-    RedisModule_ThreadSafeContextLock(ctx);
-    int ret = RedisAI_Initialize(ctx);
+    int ret = RedisAI_Initialize(staticCtx);
```

Comment:
We need to verify here that the version is greater than 6.0.7 (IIRC), otherwise we will crash here.

Reply:
@alonre24 what about this one?